### ISSUE TITLE
Telegram Notify : TypeError: Cannot read property 'en' of undefined

### ISSUE THREAD

**Issue by DutchmanNL** [ioBroker/ioBroker.link#9]
```

2019-12-04 22:04:22.271 - debug: link.0 (2203) AccessClient: setting up reverse proxy
--
2019-12-04 22:04:22.271 - debug: link.0 (2203) no UDP devices found. Skipping setting up SSF tunnel
2019-12-04 22:04:22.272 - debug: link.0 (2203) SSH Client :: connecting to xxx.xxx.xxx.xxx
2019-12-04 22:04:22.574 - debug: link.0 (2203) SSH Client :: ready
2019-12-04 22:04:22.575 - debug: link.0 (2203) AccessClient: accepting connection request: 40242442-f23b-4b88-a438-3a08f3028304 from rdrozda@hotmail.com
2019-12-04 22:04:22.597 - debug: link.0 (2203) SSH Client :: listening for connections on 79.143.182.215:5006
2019-12-04 22:04:22.659 - debug: link.0 (2203) SSH Client :: listening for connections on 79.143.182.215:5007
2019-12-04 22:04:22.732 - debug: link.0 (2203) AccessClient: connection request:xxx from xxx@xxx.uk accepted
2019-12-04 22:04:22.732 - debug: link.0 (2203) AccessClient: setting up polling for request status change
2019-12-04 22:04:22.734 - error: link.0 (2203) uncaught exception: Cannot read property 'en' of undefined
2019-12-04 22:04:22.734 - error: link.0 (2203) TypeError: Cannot read property 'en' of undefined
at _0x23d9e1 (/opt/iobroker/node_modules/iobroker.link/main.js:3:5798)
at Request._callback (/opt/iobroker/node_modules/iobroker.link/main.js:3:12375)
at Request.self.callback (/opt/iobroker/node_modules/request/request.js:185:22)
at Request.emit (events.js:198:13)
at Request. (/opt/iobroker/node_modules/request/request.js:1161:10)
at Request.emit (events.js:198:13)
at IncomingMessage. (/opt/iobroker/node_modules/request/request.js:1083:12)
at Object.onceWrapper (events.js:286:20)
at IncomingMessage.emit (events.js:203:15)
at endReadableNT (_stream_readable.js:1143:12)
2019-12-04 22:04:22.735 - debug: link.0 (2203) can not stop ssf: ssf is not running
2019-12-04 22:04:22.757 - debug: link.0 (2203) SSH Client :: stopped listening for connections on 79.143.182.215:5006
2019-12-04 22:04:22.757 - info: link.0 (2203) terminating
2019-12-04 22:04:22.758 - info: link.0 (2203) Terminated (NO_ERROR): Without reason
2019-12-04 22:04:22.778 - debug: link.0 (2203) SSH Client :: stopped listening for connections on 79.143.182.215:5007
2019-12-04 22:04:22.802 - debug: link.0 (2203) SSH Client :: disconnected
2019-12-04 22:04:22.803 - debug: link.0 (2203) {"protocol":"TCP","connected":false}
2019-12-04 22:04:22.803 - debug: link.0 (2203) {"protocol":"TCP","connected":false}
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: TypeError: Cannot read property 'en' of undefined
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at _0x23d9e1 (/opt/iobroker/node_modules/iobroker.link/main.js:3:5798)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at Request._callback (/opt/iobroker/node_modules/iobroker.link/main.js:3:12375)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at Request.self.callback (/opt/iobroker/node_modules/request/request.js:185:22)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at Request.emit (events.js:198:13)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at Request. (/opt/iobroker/node_modules/request/request.js:1161:10)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at Request.emit (events.js:198:13)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at IncomingMessage. (/opt/iobroker/node_modules/request/request.js:1083:12)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at Object.onceWrapper (events.js:286:20)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at IncomingMessage.emit (events.js:203:15)
2019-12-04 22:04:23.273 - error: host.ioBroker Caught by controller[0]: at endReadableNT (_stream_readable.js:1143:12)
2019-12-04 22:04:23.273 - info: host.ioBroker instance system.adapter.link.0 terminated with code 0 (NO_ERROR)
2019-12-04 22:04:23.274 - info: host.ioBroker Restart adapter system.adapter.link.0 because enabled
```

**Comment by DutchmanNL** [comment]
tried modifying several settings but can't find the RootCasue of "TypeError: Cannot read property 'en' of undefined"

**Comment by DutchmanNL** [comment]
looks like it has something to do with the language settings, changing to DE results in : 

```
2019-12-06 08:48:45.176  - error: link.0 (29194) uncaught exception: Cannot read property 'de' of undefined
2019-12-06 08:48:45.176  - error: link.0 (29194) TypeError: Cannot read property 'de' of undefined
    at _0x23d9e1 (/opt/iobroker/node_modules/iobroker.link/main.js:3:5798)
    at Request._callback (/opt/iobroker/node_modules/iobroker.link/main.js:3:12375)
    at Request.self.callback (/opt/iobroker/node_modules/request/request.js:185:22)
    at Request.emit (events.js:198:13)
    at Request.<anonymous> (/opt/iobroker/node_modules/request/request.js:1161:10)
    at Request.emit (events.js:198:13)
    at IncomingMessage.<anonymous> (/opt/iobroker/node_modules/request/request.js:1083:12)
    at Object.onceWrapper (events.js:286:20)
    at IncomingMessage.emit (events.js:203:15)
    at endReadableNT (_stream_readable.js:1143:12)
2019-12-06 08:48:45.222  - info: link.0 (29194) terminating
2019-12-06 08:48:45.222  - info: link.0 (29194) Terminated (NO_ERROR): Without reason
2019-12-06 08:48:45.737  - error: host.ioBroker Caught by controller[0]: TypeError: Cannot read property 'de' of undefined
2019-12-06 08:48:45.737  - error: host.ioBroker Caught by controller[0]:     at _0x23d9e1 (/opt/iobroker/node_modules/iobroker.link/main.js:3:5798)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at Request._callback (/opt/iobroker/node_modules/iobroker.link/main.js:3:12375)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at Request.self.callback (/opt/iobroker/node_modules/request/request.js:185:22)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at Request.emit (events.js:198:13)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at Request.<anonymous> (/opt/iobroker/node_modules/request/request.js:1161:10)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at Request.emit (events.js:198:13)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at IncomingMessage.<anonymous> (/opt/iobroker/node_modules/request/request.js:1083:12)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at Object.onceWrapper (events.js:286:20)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at IncomingMessage.emit (events.js:203:15)
2019-12-06 08:48:45.738  - error: host.ioBroker Caught by controller[0]:     at endReadableNT (_stream_readable.js:1143:12)
2019-12-06 08:48:45.738  - info: host.ioBroker instance system.adapter.link.0 terminated with code 0 (NO_ERROR)
```

**Comment by gh-god** [comment]
yes, we're aware of the issue. please try to disable telegram notifications for now, that should help

**Comment by DutchmanNL** [comment]
Yes, now it works thank you !

will keep the issue open, the notify and approval by telegram is a nice feature

**Comment by gh-god** [comment]
should be fixed in 0.5.7

**Comment by DutchmanNL** [comment]
> should be fixed in 0.5.7

works including confirmation message if set, thank you !
